addons/etcd/values.yaml

```diff
@@ -4,6 +4,42 @@ fullnameOverride: ""
 
 clusterDomain: ".cluster.local"
 
+# volume backup
+dataMountPath: /var/run/etcd
+dataDir: /var/run/etcd/default.etcd
+
+tlsMountPath: /etc/pki/tls
+
+# a copy version of /etc/etcd/etcd.conf but add some modification
+tmpConfigPath: /var/run/etcd/etcd.conf
+
+# These TLS settings are used to configure two kind of TLS for etcd
+# They will only take effect when 'etcd-cluster.Values.tlsEnable' is set to true
+tls:
+  client: true
+  peer: true
+
+
+# define a threshold to check if backup total key is too low
+# '8' is the initial count for a 3-replica etcd cluster
+# consider increasing this value when dealing with larger etcd clusters
+backupKeyThreshold: 8
```

Comment:
so if it's an empty etcd, backup will fail?

Reply:
If cluster replicas is set to 3, the initial number of etcd backup key is 8, it's no smaller than backupKeyThreshold, so backup and restore will run as we expected.

However, if the replicas is reduced to 1, the initial number of etcd backup key is 6, the backup will fail unless new keys are added.

Maybe change default backupKeyThreshold to 6 is better, so backup and restore will run well in any replicas.